tools/checkTheme.js

```diff
@@ -167,37 +167,36 @@ function check_group(group, rules) {
     return [scope, has_rule(selector, rules), skips_rule(selector, rules)];
   });
 
-
   const doesNotSupport = has_rules.map(x => x[1]).includes(false);
   const skipped = has_rules.find(x => x[2]);
   if (doesNotSupport || skipped) {
-    console.log(group.name.yellow);
```

Comment:
`group.name` is the string that prototype was extended with the `yellow` function. 
This is unreadable and is very bad practice.